package.json

```
@@ -43,8 +43,8 @@
     "reconnect-core": "^1.3.0"
   },
   "peerDependencies": {
-    "winston": "*",
-    "winston-transport": "*"
+    "winston": "^3.2.1",
+    "winston-transport": "^3.2.1"
```

https://github.com/rapid7/r7insight_node/blob/387d8d579dce9ca12d32bc8620a9c034266ca7e6/package.json#L61
```suggestion
    "winston-transport": "^4.3.0"
```

---

Any particular reason for the bump to `^4.3.0`? Current dev dependencies(and as such, our tests) use `^3.2.1`, and I'd prefer to limit the scope of this PR if possible instead of also doing a winston upgrade

---

the current version is `"winston-transport": "4.3.0"`

---

Oh, so it is. My bad, I read this as `winston` rather than `winston-transport`. I'll update

---

Done

---

ughhhh, i'm confused , here i'm talking about `winston-transport`, not `winston`

we pin these to the versions we have in package.json, which are
```json
"winston": "^3.2.1",
"winston-transport": "4.3.0"
```